src/url-state.js

```diff
@@ -47,3 +53,51 @@ export function updateURL (params = {}) {
   const queryString = addNewParam(params)
   window.history.replaceState({}, null, locationPrefix + queryString)
 }
+
```

Comment:
In retrospect, I think we should do the following:

- move `url-state.js` to `lib/url-state.js`. It's a pretty generic set of url-state functions.
- put `initApp` and `initRoute` into a separate module (maybe called `init.js`), along with the imports that it needs, and it can be left in the `src` directory. It should also import the functions that it needs from `url-state`. This way we can write tests for `url-state.js` later that doesn't rely on the existence of a Redux store.